AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientBaseTests.swift

```diff
@@ -57,8 +57,8 @@ class AWSMobileClientBaseTests: XCTestCase {
     }
```

Comment:
nit: I think we've been using "XXXTestBase" as a naming convention for test superclasses, to differentiate it from actual tests, which end in "Tests"

Reply:
Updated the class name